.github/workflows/reusable-linters.yml

```diff
@@ -61,7 +61,7 @@ jobs:
       run: |
         make lint
     - name: Send coverage data to Codecov
-      uses: codecov/codecov-action@v4
+      uses: codecov/codecov-action@v3
```

Comment:
This doesn't fail. No need.

Reply:
What about the consistency?

Reply:
`v4` doesn't look like a stable release so far.

Reply:
If GitHub didn't want people to use different versions of software, the wouldn't have a version spec in every place.

Reply:
I've removed the unnecessary downgrade.